client/lib/feed-stream-store/feed-stream.js

```diff
@@ -294,29 +295,29 @@ assign( FeedStream.prototype, {
 
 		this.onUpdateFetch( params );
 		debug( 'polling for updates' );
-		this.fetcher( params, function( error, data ) {
+		this.fetcher( params, ( error, data ) => {
 			debug( 'poll response received', error, data );
 			this.isFetchingUpdates = false;
 			FeedStreamActions.receiveUpdates( this.id, error, data );
-		}.bind( this ) );
-	},
+		} );
+	}
 
 	/**
 	 * Returns a list of sites this url was cross posted to.
 	 * @param {string} postURL - i.e.: https://developer.wordpress.com/blog/2015/10/08/this-is-just-to-say/
 	 * @returns {array} - i.e. [ { siteURL: http://dailypost.wordpress.com/, siteName: "+dailypost" } ]
 	 */
-	getSitesCrossPostedTo: function( postURL ) {
+	getSitesCrossPostedTo( postURL ) {
 		return this.xPostedByURL[ postURL ];
-	},
+	}
 
 	/**
 	 * If possible, returns the post metadata from a given response.
 	 * @param {object} postWrapper - response with post metadata
 	 * @returns {*} - null or post object
 	 * @private
 	 */
-	_getPostFromMetadata: function( postWrapper ) {
+	_getPostFromMetadata( postWrapper ) {
```

Comment:
Can we remove the initial `_` from the method name?

(mentioned in https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md)